cob_base_velocity_smoother/mainpage.dox

```diff
@@ -2,9 +2,9 @@
 \mainpage
 \htmlinclude manifest.html
 
-\b cob_base_velocity_smoother is ... 
+\b cob_base_velocity_smoother is ...
```

Comment:
Indeed!
E.g. `cob_cartesian_controller` does not have a `mainpage.dox` and the API site looks like [this](http://docs.ros.org/indigo/api/cob_cartesian_controller/html/)
So I'm going to remove all mainpage.dox files that only have autogenerated content....